### PR TITLE
Add method to produce inputs for Kafka CLI

### PR DESCRIPTION
**Pull request by pacospace**
## Related Issues and Dependencies

Depends-On: https://github.com/thoth-station/workflow-helpers/pull/20
Related-To: https://github.com/thoth-station/solver/issues/579

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add workflow helper to parse solver inputs and create inputs to send SolvedPackage Kafka message.

### CODE REVIEW

File: parse_solver_inputs.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import logging
+import json
+
+
+_LOGGER = logging.getLogger("thoth.parse_solver_inputs")
+
+def parse_solver_inputs():
+    """Parse solver inputs and produce inputs for Kafka message."""
+    solver_name = os.environ["THOTH_SOLVER_NAME"]
+    packages = os.environ["THOTH_SOLVER_PACKAGES"]
```
Comment (KPostOffice) [inline parse_solver_inputs.py:30]: let's use `package` and `index` since we only accept a single one of each

Comment (pacospace) [reply parse_solver_inputs.py]: Thanks, updated!

File: parse_solver_inputs.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    package = os.environ["THOTH_SOLVER_PACKAGES"]
+
+    # TODO: Solver can be scheduled also with list of indexes!
+    indexes = os.environ["THOTH_SOLVER_INDEXES"]
+
+    index_url = indexes[0]
+
+    package_inputs = package.split("===")
+    message_input = {
+        "package_name": {"type": "str", "value": package_inputs[0]}
```
Comment (KPostOffice) [inline parse_solver_inputs.py:39]: missing commas

Comment (pacospace) [reply parse_solver_inputs.py]: Updated in https://github.com/thoth-station/workflow-helpers/pull/25/commits/c2eded39320264a7313a555d9a629bf132190bf3

File: app.sh [hunk 3]
```diff
@@ -18,4 +18,6 @@ elif [ "$THOTH_WORKFLOW_TASK" = "qeb_hwt" ]; then
     exec python3 qeb_thamos_advise.py
 elif [ "$THOTH_WORKFLOW_TASK" = "download_package" ]; then
     exec python3 download_py_package.py
+elif [ "$THOTH_WORKFLOW_TASK" = "parse_solver_inputs"]; then
```
Comment (KPostOffice) [inline app.sh:21]: need space here after "parse_solver_inputs"

**Pull request merged**